R/games.R

```diff
@@ -1425,7 +1424,7 @@ sample_pref <- function(nodes, types, type.dist = rep(1, types),
                         pref.matrix = matrix(1, types, types),
                         directed = FALSE, loops = FALSE) {
   if (nrow(pref.matrix) != types || ncol(pref.matrix) != types) {
-    stop("Invalid size for preference matrix")
+    cli::cli_abort("Invalid size for preference matrix")
```

Comment:
Rephrase with must, maybe add a hint to read the manual?
```suggestion
    cli::cli_abort(c(
      "{.arg pref.matrix} must have {.arg types} rows and columns.",
      i = "See {.fun igraph::sample_pref}'s manual."
     ))
```